apps/auth/main.py

```
@@ -22,17 +24,25 @@ def frontendinfo():
     }
 
 @app.post("/login")
-def login(data: models.LoginData):
+def login(response: Response, data: models.LoginData):
```

Ruff (0.8.2)

Missing return type annotation for public function `login`

(ANN201)